suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml

```diff
@@ -160,20 +160,38 @@ tests:
       destroy-cluster: false
       abort-on-fail: true
   - test:
-      name: Upgrade ceph
-      desc: Upgrade cluster to latest version
-      module: test_cephadm_upgrade.py
-      polarion-id: CEPH-83573791
-      config:
-        command: start
-        service: upgrade
-        base_cmd_args:
-          verbose: true
-        benchmark:
-          type: rados                      # future-use
-          pool_per_client: true
-          pg_num: 128
-          duration: 10
-        verify_cluster_health: true
-      destroy-cluster: false
-      abort-on-fail: true
+      name: Parallel run
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: rados_bench_test
```

Comment:
IMO, we can take away RADOSbench test, Since its executed along with upgrade test.

Reply:
if possible, maybe we can add RGW data OPs.